main.cpp

```diff
@@ -404,15 +575,40 @@ struct ElectricGuitar
     String stockRoundWound090;
     Pickup stockSingleCoil;
 
-    //returns how much money will get from selling this guitar
     float sellMoney(float totallValue = 0, float askingPrice = 0, float buyerBudget = 0);
     void playRockMusic(SongWriter rockWriterA, std::string songName = "Hell's Bell");
-    
-    //returns if the guitar is currently collecting dust
-    bool collectDust(int useTimesInRecentWeek = 0);
+    bool usageCheck(int useTimesInRecentWeek = 0, bool collectDust = false);
 };
 
+float ElectricGuitar::sellMoney(float totalValue, float askingPrice, float buyerBudget)
+{
+    if(buyerBudget > askingPrice)
+    {
+        totalValue = buyerBudget;
+    }
+    return totalValue;
+}
+
+void ElectricGuitar::playRockMusic(SongWriter rockWriterA, std::string songName)
+{
+    if(songName == "Hell's Bell")
+    {
+        rockWriterA.increaseFollower();
+    }
+}
 
+bool ElectricGuitar::usageCheck(int useTimesInRecentWeek, bool collectDust)
+{
+    if(useTimesInRecentWeek == 0)
+    {
+        collectDust = true;
+    }
+    else
+    {
+        collectDust = false;
+    }
+    return collectDust;
```

Comment:
Don't pass in an argument if you're not going to use it.
just declare `collectDust` as a local variable and remove it as a function parameter.
- [ ] revise